internal/pproflogging/pproflogging.go

```diff
@@ -281,7 +328,10 @@ func DumpPem(bs []byte, types string, wrt *os.File) error {
 		return fmt.Errorf("could not write PEM: %w", err2)
 	}
 
-	// did not get a read error.  file ends in newline
+	if err0 != nil {
```

Comment:
What if `wrt.Write()` failed on this line above ?

```
for err1 == nil && err2 == nil {
    :
    :
    _, err2 = wrt.Write(ln)
```

The for loop will break out and we'll leave the above goroutine deadlocked ? Maybe I am not reading the code correctly... ?

Reply:
thank you for fixing